### ISSUE TITLE
Insufficient Color Contrast

### ISSUE THREAD

**Issue by BasaltBot** [basaltinc/knapsack#248]
The light grey and iron grey text color has failing contrast over white background.

The black basalt logo has failing contrast against the blue header- Suggest using a white text logo.

The yellow focus color on the <aside> tray toggle has failing contrast over white background.

The focus color for elements has failing contrast over white background making tab navigation difficult.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1126343145187461/1126484338969281)
